pennylane/operation.py

```diff
@@ -542,6 +542,72 @@ def eigvals(self):
         """
         return self._eigvals(*self.parameters)
 
+    @staticmethod
+    def compute_terms(*params, **hyperparams):
+        r"""Static method to define the representation of this operation as a linear combination.
+
+        Each term in the linear combination is a pair of a scalar
+        value :math:`c_i` and an operator :math:`O_i`, so that the sum
+
+        .. math:: O = \sum_i c_i O_i
+
+        constructs this operator :math:`O`.
+
+        A ``NotImplementedError`` is raised if no representation by terms is defined.
+
+        .. note::
+            This method gets overwritten by subclasses to define the linear combination representation
+            of a particular operator.
+
+        Args:
+            params (list): trainable parameters of this operator, as stored in the ``parameters`` attribute
+            hyperparams (dict): non-trainable hyperparameters of this operator, as stored in the
+                ``hyperparameters["ops"]`` attribute
```

Comment:
what do you mean by the `["ops"]` part here?

Reply:
Shouldn't this just be
```
hyperparams (dict): non-trainable hyperparameters of this operator, as stored in the
``hyperparameters`` attribute
```

Reply:
Whoops, I got confused which docstring it was